agent/agent.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2021, OpenTelemetry Authors
```

Comment:
Let's use `Copyright The OpenTelemetry Authors` to avoid the need to update the year.